staging/src/k8s.io/kubectl/pkg/cmd/create/create_cronjob.go

```diff
@@ -193,7 +196,10 @@ func (o *CreateCronJobOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, a
 // Run performs the execution of 'create cronjob' sub command
 func (o *CreateCronJobOptions) Run() error {
 	if o.Client != nil {
-		cronJob := o.createCronJob()
+		cronJob, err := o.createCronJob()
```

Comment:
What if instead of changing this function signature, you do something like:

```golang
labels, err := cmdutil.ParseLabel(o.Labels)
if err != nil {
     return err
}
if o.Client != nil {
  cronJob := o.createCronJob()
  cronjob.Labels = labels
[...]
cronJobBeta := o.createCronJobBeta()
cronJobBeta.Labels = labels
```